src/udev/memory_id/types.h

```diff
@@ -0,0 +1,63 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
i think we don't need this header, do we? it's only used by a single .c file after all. move everything in there?